### ISSUE TITLE
'You must feed a value for placeholder' when using mytcav.run

### ISSUE THREAD

**Issue by DemonHugo** [tensorflow/tcav#63]
Hi there,
I`m using the TCAV with my project. I modified the model wrapper for my project and it worked when I do
`act_generator = act_gen.ImageActivationGenerator(mymodel, source_dir, activation_dir, max_examples=200)`

However, it failed when I using `results = mytcav.run(run_parallel=False)`
The error is `You must feed a value for placeholder tensor 't_input' with dtype float and shape [?,?,?,3]` which 't_input' is defined at model.py def import_graph. 

In addition, there is my wrapper code:

`class MyWrapper_public(PublicImageModelWrapper):
  def __init__(self, sess, model_saved_path, labels_path):
    self.image_value_range = (0, 1)
    image_shape_v3 = [224, 224, 3]
    endpoints = dict(
        input='x:0',
        dropout='Placeholder:0',
        is_train='Placeholder_1:0',
        logit='fc2/BatchNorm/Reshape_1:0',
        prediction='Softmax:0'
        # pre_avgpool='mixed_10/join:0',
        # logit_weight='softmax/weights:0',
        # logit_bias='softmax/biases:0',
    )
    self.sess = sess
    super(MyWrapper_public, self).__init__(sess,
                                                  model_saved_path,
                                                  labels_path,
                                                  image_shape_v3,
                                                  endpoints,
                                                  scope='my')
    self.model_name = 'MyWrapper'`
 
Thanks

**Comment by DemonHugo** [comment]
And if I feed 't_input' values in def get_gradient in 'model.py', The speed will slow down dramatically.
With 'max_examples = 20' and 'num_random_exp=10', it spent me almost 5 hours.
and my environment are:
tensorflow-gpu 1.14,
CPU-intel-9700KF,
GPU-Nvidia-2060s

**Comment by DemonHugo** [comment]
It turns out that I used wrong bottleneck tensors.
I used tf.slim and BN layers in my model, and there are two FuseBatchNormal ops in one layer. Once I changed the bottleneck tenser, it works.
